source/develop/release-management/features/network/ipv6_subnets.html.md

```diff
@@ -152,21 +366,138 @@ provision the following networking API entities, represented in yaml:
+| DNS recursive name server |       DHCP      |       DHCP       |
+|     Domain Search List    |       DHCP      |       DHCP       |
+|       Network Prefix      |       None      |        RA        |
+|         IP address        |       DHCP      |  Auto generated  |
+|   Network Prefix Length   |        RA       |        RA        |
+|            MTU            |        RA       |        RA        |
+|      Default Gateway      |        RA       |        RA        |
+
+### Supported use cases per dhcpv6 configuration
+
+| Tables                  |           dhcpv6-stateful           | dhcpv6-stateless |
+|-------------------------|:-----------------------------------:|:----------------:|
+| fixed IPs               |              supported              |   not supported  |
+| allowed prefix lengths  |            64 <= X <= 128           |      64 only     |
+| ipv6 privacy extensions |            not supported            |   not supported  |
+| isolated subnet         | if guest routes manually configured |   not supported  |
+
+A brief explanation of each of the use cases described in the tables can be
+found below.
+
+#### Fixed IPs
+
+Fixed IPs are not supported on dhcpv6-stateless since the client itself
+auto-generates the IP address based on the advertised network prefix.
+This means the IP address ***is not*** sent in the DHCPv6 Reply message, thus
+making the guest VM IP address configuration impossible.
+
+#### Allowed prefix lengths
+
+The [slaac](https://tools.ietf.org/html/rfc4862) RFC indicates that all network
+prefixes **are** 64 bit long. Since on DHCPv6 stateless the IP prefix
+configuration is done as in slaac, the IPv6 subnets having dhcpv6_stateless
+address mode are thus limited to /64 prefixes.
+
+On DHCPv6_stateful, the IP address is sent on the DHCPV6 Reply message, and the
+network prefixes - and lengths - are communicated through RA messages.
+
+Thus, the guest is able to configure its addresses - and routes - to communicate
+with subnets having longer prefixes.
+
+There is a caveat though - dynamic IP addresses cannot be used, meaning they
+will have to be configured through the networking API [fixed IPs](https://developer.openstack.org/api-ref/network/v2/?expanded=create-subnet-detail,create-port-detail#create-port)
+create port method - specifying both the subnet id, and the IP address.
+
+#### IPv6 privacy extensions
+
+Privacy extensions are defined in [rfc](https://tools.ietf.org/html/rfc4941).
+They can **only** be applied to stateless address modes, and OVN does not
+implement it.
+
+The only IP address assignment mechanisms currently implemented in OVN use the
+MAC address to configure the host part of the IPv6 address.
+
+#### isolated subnet
+
+Isolated subnets - not attached to OVN logical routers - can be achieved if the
+address mode is set to dhcpv6_stateful. The caveat here is that the user/admin
+has to configure the guest VM routes manually, since there is no router to send
+the RA messages.
 
```

Comment:
You can mention that link-local addresses could be used, which does not require the manual configuration of routes, but the knowledge of the link-local address of the communication partner. The link-local address is shown in oVirt UI.

Reply:
I'm not being able to establish this, hence I'm not mentioning it.

Even when I ping the multicast all nodes address (ff02::1%<if name>) I only the the client's reply.

The NDP flows I see do not feature any link local addresses, that's probably why.

Reply:
I've tested this and couldn't get it to work - https://imgur.com/a/yFskQU9

Reply:
Would it work without subnet for you?

Reply:
No, it didn't work.